docs/TELEMETRY.md

```diff
@@ -23,35 +23,75 @@
 
 The STUDY SPECIFIC ENDINGS this study supports are:
 
+I don't think there are any ways this study can end besides being manually disabled or unenrolled? I think survey should be shown to all enrollees at unenrollment.
+
```

Comment:
Should we make a section to define/describe the treatment and control groups?

Reply:
Good point, I'll add that